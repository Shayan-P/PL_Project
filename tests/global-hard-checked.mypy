checked

def f():
    global a;
    print(a + [1, 2, 3]);
;

a = [2, 3];
f();