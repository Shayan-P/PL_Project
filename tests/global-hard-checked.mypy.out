type-checked!
'(2 3 1 2 3)
'finished
